lua/session_manager/init.lua

```diff
@@ -13,13 +14,24 @@ function session_manager.load_last_session(bang)
   end
 end
 
+function session_manager.load_current_dir_session(bang)
+  local session_name = utils.dir_to_session_filename(vim.loop.cwd())
+  if Path:new(session_name):exists() then
+    utils.load_session(session_name, bang and #bang ~= 0)
+  end
+end
+
 function session_manager.save_current_session()
   utils.save_session(utils.dir_to_session_filename())
 end
 
 function session_manager.autoload_session()
   if config.autoload_last_session and vim.fn.argc() == 0 then
-    session_manager.load_last_session()
+    if config.autoload_current_session then
+      session_manager.load_current_session()
```

Comment:
You probably forgot to rename:

```suggestion
      session_manager.load_current_dir_session()
```

Reply:
yes, i noticed it too, fixed now